spec/features/comments/user_can_comment_on_a_job_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe "User visits a specific job and can add a comment" do
+  scenario "user inputs a comment" do
```

Comment:
be sure to test the sad path here - what happens when a user enters invalid data?

also - this is a lot for one test. consider breaking this up